cfme/tests/v2v/test_csv_import.py

```diff
@@ -0,0 +1,182 @@
+import fauxfactory
+import pytest
+import tempfile
+
+from cfme.fixtures.v2v import _form_data
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_VERSION
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.generators import random_vm_name
+from cfme.utils.wait import wait_for
+
+pytestmark = [
+    pytest.mark.ignore_stream('5.8'),
+    pytest.mark.provider(
+        classes=[RHEVMProvider],
+        selector=ONE_PER_VERSION
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_VERSION,
+        fixture_name='second_provider'
+    )
+]
+
+
+@pytest.fixture(scope="function")
+def ims_map(appliance, v2v_providers):
```

Comment:
**Question**: Should we introduce `ims` word in codebase? I believe we have been mostly sticking to v2v, so we should use same names.

Reply:
Agreed! Adding `v2v_map` there. :+1:

Reply:
Agreed! Adding `infra_map` there. :+1: